src/Microsoft.ML.Data/DataView/Transposer.cs

```diff
@@ -666,12 +668,14 @@ private void EnsureValid()
                                         values = _values[s];
                                     }
                                     var rbuff = _rbuff[pair.Value];
+                                    var rbuffValues = rbuff.GetValues();
```

Comment:
>rbuff.GetValues() [](start = 54, length = 17)

I don't know how heavy AsSpan operations are, so maybe it's just minor thing, but we call GetValues and GetIndices for each _rbuff[r] in loop above, can we just reuse them?

Reply:
They are fairly light-weight, but in general you don't want to keep doing it over and over in a tight loop.

However, here we have a `VBuffer[]`. You can't have an array/collection of Spans. So we need to first get the `VBuffer` we are interested in, and then grab its value span. We can't do that above the loop.